src/main/java/org/testng/internal/Parameters.java

```diff
@@ -102,7 +102,7 @@
   public static Object[] createConfigurationParameters(Method m,
       Map<String, String> params,
       Object[] parameterValues,
-      @Nullable ITestNGMethod currentTestMethod,
```

Comment:
I'm fine removing TestNG's own `@Nullable` but please keep the annotations in the code base. If anything, I'd like to see more uses of `@Nullable` and `@NonNull`, which have a lot of benefits (Kotlin interop, IDE checking things for you, etc...).